tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>can you avoid to call <code>create</code> and just create it the first time the attribute is accessed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>The create method has options such as <code>dtype</code>,  <code>shape</code>,  etc.  I am using defaults here, but there is no way to guess what a user wants to create when they do <code>ob.detdata["foobar"]</code>.   Maybe they are creating detector data of type <code>np.uint16</code>  with 5 values per sample.  The names used for signal, flags, etc are controlled by the operators that create those fields.  So there are defaults, but in principal a user could build a self-consistent pipeline with other names for those quantities.</p>

Reply:
you can get `dtype` and `shape` from the argument they are trying to set

Reply:
<p>Notebook updated to show 3 different ways of creating new buffers (explicit method, assign from full array, assign from replicated array for one detector).</p>